src/base/constants.py

```diff
@@ -44,3 +44,16 @@ def _IsDevAppServer():
 DEBUG = IS_DEV_APPSERVER
 
 TEMPLATE_DIR = os.path.sep.join([os.path.dirname(__file__), '..', 'templates'])
+
+AUTHORIZATION_METHOD = 'basic_auth'
+
+# Email allowed to log in (only when AUTHORIZATION_METHOD == 'google')
+EMAIL_WHITELIST = {
+    'unit9.com',
+    'test@example.com'
+}
+
+ADMIN_DATA = ('login_name', 'password')
+auth_key = base64.b64encode(':'.join(ADMIN_DATA))
```

Comment:
- NOT the right place to encode random constants!
- Best if you'd instead DEcode what user supplied and compare decoded values